ipatests/test_integration/test_commands.py

```diff
@@ -1170,3 +1170,35 @@ def test_ipa_adtrust_install_with_locale_issue8066(self):
             assert msg2 not in result.stderr_text
         finally:
             bashrc_backup.restore()
+
+    def test_login_wrong_password(self):
+        """Test ipa user login with wrong password
+
+        When ipa user login to machine using wrong password, it
+        should log proper message in /var/log/secure
```

Comment:
Please update the comment, the message is not checked in /var/log/secure on fedora. I would simply write "it should log proper message".